doc/source/whatsnew/v0.16.2.txt

```diff
@@ -25,6 +25,8 @@ New features
 Other enhancements
 ^^^^^^^^^^^^^^^^^^
 
+- Fix undesired UX behavior of DataFrame output in IPython Notebook. (:issue:`10231`)
```

Comment:
I might say: "Removed duplicate scroll bars in DataFrame HTML representation in IPython Notebook" (just to be a little more descriptive)